docs/product/issues/issue-details/index.mdx

```diff
@@ -6,29 +6,44 @@ description: "Learn how to navigate the Issue Details page to help you efficient
 
 The **Issue Details** page helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific event that's part of an issue. The top panel and right-hand sidebar of the page display a summary of all the events grouped together in this issue:
 
-![Issue details](./img/issue_details_page.png)
+![Issue details](./img/issue-details-page.png)
 
 You can view information about a specific error event that's part of an issue on the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page. You can also navigate between the events of an issue using the event navigator.
 
 In the right hand sidebar, [sentry.io](https://sentry.io) reflects a summary that includes information such as how often the <SandboxLink scenario="oneIssue" projectSlug="react">issue</SandboxLink> has occurred in the last 24 hours and the last 30 days, as well as the last time and the first time the issue was seen. If the issue is linked to any GitHub or Jira issues, that's displayed here as well. Lastly, there is a facet map, which shows the distribution of values for the tags for all events included in the issue. All of these values are based on the environment that you've selected in the environment dropdown.
 
+## Issue Tabs
```

Comment:
Moved this section up to the top of the page since it seems like this page follows the same pattern as the issue details page, top to bottom.